fiftyone/operators/cache/decorator.py

```
@@ -0,0 +1,166 @@
+"""
+Execution cache decorator.
+
+| Copyright 2017-2025, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+from functools import wraps
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)